src/ruvsarpur.py

```diff
@@ -439,8 +452,8 @@ def runMain():
         # Store the id as already recorded 
         previously_recorded.append(item['pid'])
 
-    # Now save the list of already recorded shows back to file and exit
-    savePreviouslyRecordedShows(previously_recorded)
+        # Now save the list of already recorded shows back to file and exit
```

Comment:
Why the extra indent? This is a regression please fix. The prev recorded shows should not be saved in each loop of the for.